File: Modules/Common/LES/SIE.Web.LES/StockOrders/Commands/StockOrderMergeTimesDeleteCommand.js
SIE.defineCommand('SIE.Web.LES.StockOrders.Commands.StockOrderMergeTimesDeleteCommand', {
    extend: 'SIE.cmd.Delete',
    meta: { text: "删除", group: "edit", iconCls: "icon-DeleteEntity icon-red" },
    canExecute: function (view) {
        var parent = view.getParent();
        if (parent) {
            var current = view.getParent().getCurrent();
            if (current && current.getState() == 0) {
                if (view.getCurrent())
                    return true;
            }
        }
        return false;
    },
})

File: Modules/Common/LES/SIE.Web.LES/StockOrders/Commands/StockOrderMergeTimesDeleteCommand.cs
using SIE.Web.Command;

namespace SIE.Web.LES.StockOrders.Commands
{
    public class StockOrderMergeTimesDeleteCommand : DeleteCommand
    {
        protected override object Excute(ViewArgs args, string scope)
        {
            return true;
        }
    }
}
